automation/project/Dependencies.scala

```diff
@@ -5,20 +5,22 @@ object Dependencies {
   val akkaV = "2.5.7"
   val akkaHttpV = "10.0.10"
 
+  val serviceTestV = "0.1-f9f7b20-SNAP"
```

Comment:
i'm sure you've already got this on your list but don't forget to update this to the non-snap when you're ready